src/legacy/core_plugins/vis_type_tagcloud/public/components/__tests__/tag_cloud_visualization.js

```diff
@@ -67,10 +70,11 @@ describe('TagCloudVisualizationTest', function() {
 
   describe('TagCloudVisualization - basics', function() {
     beforeEach(async function() {
+      const visType = new BaseVisType(createTagCloudVisTypeDefinition({ colors: seedColors }));
```

Comment:
Thanks for doing 80% of our job with the legacy tests - they are on our list but we didn't get to them yet.